src/deploy/jinja2_templates/nic-configs/5_port/nic_environment_edge.j2.yaml

```diff
@@ -18,6 +18,10 @@ resource_registry:
 {%- endfor %}
 
 parameter_defaults:
+  ControllerBondInterfaceOptions: mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
+  ComputeBondInterfaceOptions: mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
+  StorageBondInterfaceOptions: mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
+  BondInterfaceSriovOptions: 802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
```

Comment:
Those should be read from ini bond settings ?

Reply:
Done, using config.get_curated_nics_settings() to fetch bonding interface options,  change in next patch set.